src/vivarium_cluster_tools/psimulate/runner.py

```diff
@@ -208,6 +223,15 @@ def main(
     # Generate a worker template that chooses a redis DB at random to connect to.
     # This should (approximately) evenly distribute the workers over the work.
     redis_urls = [f"redis://{hostname}:{port}" for hostname, port in redis_ports]
+
+    # Grab redis urls for dashboard and send them to function for popen
+    rq_urls = [
+        f"redis://{hostname}.cluster.ihme.washington.edu:{port}"
```

Comment:
The workers are okay with this change?

Reply:
I'm not sure I understand your question but this isn't doing anything to the workers it is just formatting the redis urls the way the rq-dashboard wants them.   It isn't doing anything to the workers themselves to my knowledge.

Reply:
So it's changing the workers by giving them the FQD of `.cluster.ihme.washington.edu`. I would presume that so long as the cluster's FQD doesn't change, we're ok.

Reply:
@mattkappel Are you saying that f"redis://{hostname}:{port}" resolves to "redis://{hostname}.cluster.ihme.washington.edu:{port}"? If so, then no need for an rq_urls and you can just pass in redis_urls?

Reply:
oh sorry, I misread! `rq_urls != redis_urls` But I would think what you think I said is true.